packages/preview/src/browser/markdown/style/markdown.css

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/*---------------------------------------------------------------------------------------------
```

Comment:
@svenefftinge and @akosyakov, I've changed the most of markdown.css, which includes:
- theming 
- highlighting elements with line info
- fixed table styles
- removed .vs(-dark)

Should we still have this (c) block?

Reply:
Could you share a link to the original?

Reply:
Is it this one? https://github.com/Microsoft/vscode/blob/master/extensions/markdown/media/markdown.css
Looks very different.

Reply:
But still, I think you could keep it and the eclipse IP team can check that again.

Reply:
Sorry, should be already attached. I copied from https://github.com/theia-ide/theia-extension-example/blob/master/theia-markdown/src/browser/style/markdown.css, which is a copy of 
https://github.com/Microsoft/vscode/blob/master/extensions/markdown/media/markdown.css

Reply:
That's okay.